test/test_docker_git.py

```diff
@@ -0,0 +1,11 @@
+from docker_cli._docker_git import git_clone_dockerfile
+
+from .utils import rough_dockerfile_validity_check
+
+
+def test_git_dockerfile():
+    """Performs a rough validity check of the Git dockerfile"""
+    header, body = git_clone_dockerfile(git_repo="abc/def", repo_branch="ghi")
+
+    dockerfile = f"{header}\n\nFROM %TEST_BASE%\n\n{body}"
+    rough_dockerfile_validity_check(dockerfile=dockerfile)
```

Comment:
Let's improve this test to make sure that we're checking:

(A) the git repo was actually cloned inside the image
(B) that the current working directory is now the cloned directory

Here's a simple git repo that you could use in your test: https://github.com/octocat/Hello-World